framework/utils/randomUtils.py

```diff
@@ -188,8 +188,8 @@ def randomIntegers(low, high, caller=None, engine=None):
   if isinstance(engine, np.random.RandomState):
     return engine.randint(low, high=high+1)
   elif isinstance(engine, findCrowModule('randomENG').RandomClass):
-    intRange = high - low
-    rawNum = low + random(engine=engine)*intRange
+    intRange = high - low + 1.0
+    rawNum = low + random(engine=engine)*intRange - 0.5
     rawInt = int(round(rawNum))
```

Comment:
Maybe we should switch this to floor or ceiling instead of round and that might simplify the previous lines.

Reply:
as in ```rawInt = math.floor(rawNum)``` and then I think the above can be ```rawNum = low + random(engine=engine)*intRange```

Reply:
```
>>> import math
>>> low = 3
>>> high = 6
>>> intRange = high - low + 1.0
>>> intRange
4.0
>>> low + 0*intRange
3.0
>>> math.floor(low + 0*intRange)
3
>>> math.floor(low + 0.99999*intRange)
6
>>> low + 0.99999*intRange
6.99996
>>> 
```

Reply:
Your method is a complementary approach. I am running your edits agianst the tests,